server/app/services/JiraReaderService.java

```diff
@@ -0,0 +1,159 @@
+    }
+
+    /**
+     * To request for JIRA ticket info page via REST API. A non-blocking call.
+     *
+     * @param ticketId ticket ID in string.
+     * @return info page encoded in JSON.
+     */
+    public CompletionStage<JsonNode> fetchTicketByApi(String ticketId) {
+
+        WSRequest request = ws.url(ConfigUtilities.getString("jira.baseUrl")
+                + ConfigUtilities.getString("jira.issueEndpoint")
+                + ticketId);
+        WSRequest complexRequest = request.setAuth(jiraAuth.username, jiraAuth.password, WSAuthScheme.BASIC);
+
+        return complexRequest.get().thenApply(WSResponse::asJson);
+    }
+
+    /**
+     * //TODO : comment this.
+     * @param response
+     * @param intent
+     * @param ticketNo
+     * @return
+     */
+    public JsonNode read(JsonNode response, String intent, String ticketNo) {
+        Boolean isSuccess = false;
+
+        switch (intent) {
+            case "IssueBrief" :
+                isSuccess = readDescription(ticketNo, response);
+                break;
+            case "IssueAssignee" :
+                isSuccess = readAssignee(ticketNo, response);
+                break;
+            case "IssueStatus" :
+                isSuccess = readStatus(ticketNo, response);
+                break;
+        }
+
+        if (isSuccess) {
+            return Json.toJson(new ResponseToClient(JiraServiceProvider.REQUEST_SUCCESS, messageToReturn));
+        } else {
+            return Json.toJson(new ResponseToClient(JiraServiceProvider.REQUEST_FAILURE,
+              ConfigUtilities.getString("error-message.issue-not-found")));
+        }
+    }
+
+    /**
+     * This method reads assignee of the ticket.
+     *
+     * @param ticketNo     is the IssueID of type string which was mentioned in the query by the user.
+     * @param responseBody is the JSON object received from JIRA Rest API.
+     * @return true if success, otherwise if no such ticket exists, false.
+     */
+    private Boolean readAssignee(String ticketNo, JsonNode responseBody) {
+        if (responseBody.get("errorMessages") != null){
+            return false;
+        } else {
+            this.messageToReturn = responseBody.get("fields").get("assignee").get("displayName").textValue()
+                    + " is working on "
+                    + ticketNo + ".";
+            return true;
+        }
+    }
+
+    /**
+     * This method reads description of the ticket.
+     *
+     * @param responseBody is the JSON object received from JIRA Rest API.
+     * @return true if success, otherwise if no such ticket exists, false.
+     */
+    private Boolean readDescription(String ticketNo, JsonNode responseBody) {
+        if (responseBody.get("errorMessages") != null) {
+            return false;
+        } else {
+            this.messageToReturn = hyperlinkTicketNo("Description of "
+                  //  + '<' + "http://google.com" +'|' +ticketNo+  '>' // replace google by the actual ticket page
```

Comment:
Please remove commented code? "//  + '<' + "http://google.com" +'|' +ticketNo+  '>' // replace google by the actual ticket page"